src/Feldspar/Onnx/Operators.hs

```diff
@@ -194,29 +194,29 @@ bcZipWith f xs ys = zipWith f (uniBCast ext xs) (uniBCast ext ys)
   where ext = unionExt (extent xs) (extent ys) 
 
 -- | Implementation of ONNX tensor addition
-onnxAdd :: (Num a, ShapelyU sh1 sh2)
-        => Attrs -> Pull sh1 a -> Pull sh2 a -> Pull (UnionShape sh1 sh2) a
-onnxAdd _ = bcAdd
+onnxAdd_2 :: (Num a, ShapelyU sh1 sh2)
```

Comment:
It would be nice if the handwritten code is readable and these are binary operators, do we really need the `_2` on them? Same question for the `_5` on `onnxBatchNormalization`, `_3` on `onnxGemm`, `_1` on `onnxFlatten`, and so on.

Reply:
The arity information is necessary because some ONNX operators can be used with different numbers of arguments. To avoid it, onnxToFeld needs to know about those operators and treat them as special cases, something I have tried to avoid, at least for now.

Reply:
Do we get a silent failure if we get the special cases wrong? If not, I would prefer an Operators.hs without extra suffixes and two additional lines (`| op elem varargOps = ""` and  `| otherwise = "_" <> show (length args)`) in OnnxToFeld.hs.